packages/backend/src/index.ts

```diff
@@ -25,7 +25,14 @@ export type ClerkOptions = CreateBackendApiOptions &
     >
   > & { sdkMetadata?: SDKMetadata; telemetry?: Pick<TelemetryCollectorOptions, 'disabled' | 'debug'> };
 
-export function createClerkClient(options: ClerkOptions) {
+// The current exported type resolves the following issue in packages importing createClerkClient
+// TS4023: Exported variable 'clerkClient' has or is using name 'AuthErrorReason' from external module "/packages/backend/dist/index" but cannot be named.
+export type ClerkClient = {
```

Comment:
Discussed offline, problematic area with types and type inference, will come back to this after v5 or next week.